docs/commands/annotatealleles_expr.md

```diff
@@ -0,0 +1,50 @@
+<div class="cmdhead"></div>
+
+<div class="description"></div>
+
+<div class="synopsis"></div>
+
+<div class="options"></div>
+
+<div class="cmdsubsection">
+### Notes:
+
+This command runs similarly to [annotatevariants expr](commands.html#annotatevariants_expr) but it dynamically splits multi-allelic sites,
+ computes each expressions on each split allele separately, and returns an **Array** with one entry per
+   non-ref allele for each expression.
+
+During the evaluation of the expressions, two additional variant annotations are accessible:
+1. `va._aIndex`: An **Int** indicating the index of the non-reference allele being evaluated
+(amongst all alleles including the reference allele, so cannot be `0` )
+2. `va._wasSplit`: A **Boolean** indicating whether this allele belongs to a multi-allelic site or not.
```

Comment:
I don't see why and underscore is necessary for these annotations.  In particular, it seems nice for an expression E to work both in:

  annotateallleles expr -c E

and

  splitmulti
  annotatevariants expr -c E